### ISSUE TITLE
Moves that trigger additional messages will basically skip the "X used Y!" text on Instant Text.

### ISSUE THREAD

**Issue by itsdarsh** [Rangi42/polishedcrystal#776]
Making this its own separate issue since the rest of #681 is resolved.

**Comment by itsdarsh** [comment]
On further consideration, this is unique enough from #899 to warrant being its own issue.

**Comment by vulcandth** [comment]
@itsdarsh Did this get fixed with the delay you added?

**Comment by itsdarsh** [comment]
Nope, delay only fixed move descriptions and autoscroll. The issue also isn't exclusive to multi-attack moves, a lot of the battle engine textboxes don't play nice with instant text.

**Comment by FredrIQ** [comment]
Fixed in [d3569b6](https://github.com/Rangi42/polishedcrystal/commit/d3569b6276d6596813fccef9c2832dadf4c0f437).
